tests/kernel/common/testcase.yaml

```diff
@@ -1,3 +1,6 @@
+common:
+  timeout: 120000
```

Comment:
isnt this a bit of a large number? this translates to ~1.5 days! We do not want that, if something else goes wrong, the test will just block everything else and CI will never complete.
Please provide additional context and why this is required now and what platforms might be affected and what has been done to verify that the added delay is not due to some bug or issue in the test or the platform itself.

Reply:
Thanks @nashif! I thought the timeout was in milliseconds, I will fix all to 120 (seconds). 
I will add more details in the PR description and commit messages if needed there too. This is a 'workaround' to allow sanitycheck to complete for up2 on these tests. I'll investigate more too so we can decide if we open a separate bug or enhancement to address the root issue(s).

Reply:
where is this extra time coming from?

if this is something that is happening for all tests on this platform, due to some unavoidable setup/flashing/teardown/bootup time or whatever, it would be better to add an option in board yaml to apply a constant offset to all timeouts instead of modifying individual test timeouts to work around it

Reply:
My next step is to add timestamps printing out for the test(s) as Anas suggested. Maybe this could help us find out if its related to setup/flashing/bootup time? I'll also try bisecting to see when/why this changed behavior to start failing in sanitycheck.

Reply:
I updated the description, hoping it helps give context. Please let me know your thoughts.